src/main/java/com/ivan/salesapp/services/OrderService.java

```diff
@@ -0,0 +1,71 @@
+package com.ivan.salesapp.services;
+
+
+import com.ivan.salesapp.domain.entities.Order;
+import com.ivan.salesapp.domain.models.service.OrderServiceModel;
+import com.ivan.salesapp.repository.OrderRepository;
+import com.ivan.salesapp.repository.ProductRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService implements IOrderService {
+
+    private final OrderRepository orderRepository;
+    private final ProductRepository productRepository;
+    private final UserService userService;
+    private final ModelMapper modelMapper;
+
+    @Autowired
+    public OrderService(
+            OrderRepository orderRepository,
+            ProductRepository productRepository,
+            UserService userService,
+            ModelMapper modelMapper
+    ) {
+        this.orderRepository = orderRepository;
+        this.productRepository = productRepository;
+        this.userService = userService;
+        this.modelMapper = modelMapper;
+    }
+
+    @Override
+    @Transactional
+    public void createOrder(OrderServiceModel orderServiceModel) {
+        orderServiceModel.setRegisterDate(LocalDateTime.now());
+
+        this.orderRepository.saveAndFlush(this.modelMapper.map(orderServiceModel, Order.class));
+    }
+
+    @Override
+    public List<OrderServiceModel> findAllOrders() {
+        List<Order> orders = this.orderRepository.findAll();
+
+        return orders
+                .stream()
+                .map(o -> this.modelMapper.map(o, OrderServiceModel.class))
+                .collect(Collectors.toList());
+    }
```

Comment:
return this.orderRepository.findAll() // 
           .stream() //
           .map(o -> this.modelMapper.map(o, OrderServiceModel.class)) //
           .collect(toList()); -> static import for Collections.toList()